it/test/uk/gov/hmrc/tradergoodsprofilesdatastore/repositories/ProfileRepositorySpec.scala

```diff
@@ -90,4 +94,54 @@ class ProfileRepositorySpec
 
   }
 
+  ".updateEori" - {
+
+    "must update profile when there is a one" in {
+      val newEori      = "new-eori"
+      insert(profileResponse).futureValue
+      val updateResult = repository.updateEori(profileResponse.eori, newEori).futureValue
+
+      val result = repository.get(newEori).futureValue
+
+      updateResult mustEqual true
+      result.value.eori mustEqual newEori
+    }
+
+    "must not update profile when there is none" in {
+      val newEori      = "new-eori"
+      val updateResult = repository.updateEori(profileResponse.eori, newEori).futureValue
+
+      val result = repository.get(newEori).futureValue
+
+      updateResult mustEqual false
+      result mustBe None
+    }
+  }
+
+  ".deleteEori" - {
+    "when there is a profile for this eori" in {
+      insert(profileResponse).futureValue
+
+      val result = repository.deleteByEori(profileResponse.eori).futureValue
+
+      result mustEqual 1
+
+    }
+
+    "when there are multiple profiles for different eori's" in {
```

Comment:
what is this test testing?

Reply:
Additional test to ensure it deletes only one profile.